packages/dejagnu.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Dejagnu < Package
+  description 'DejaGnu is a framework for running test suites on GNU tools.'
+  homepage 'https://www.ansible.com/'
+  version '1.6.1'
+  source_url 'https://ftp.gnu.org/gnu/dejagnu/dejagnu-1.6.1.tar.gz'
```

Comment:
It would be better if you used the mirror url: https://ftpmirror.gnu.org/dejagnu/dejagnu-1.6.1.tar.gz